base_headers_webkit/base_headers_data.xml

```diff
@@ -69,11 +69,11 @@
                 <td/>
             </tr>
             <tr>
-                <td>Phone: ${company.partner_id.phone or ''|entity} </td>
+                <td>${_('Phone:')} ${company.partner_id.phone or ''|entity} </td>
                 <td/>
             </tr>
             <tr>
-                <td>Mail: ${company.partner_id.email or ''|entity}<br/></td>
+                <td>${_('Mail:')} ${company.partner_id.email or ''|entity}<br/></td>
```

Comment:
I didn't. I'm not sure translations will be used there even if they are exported. The translation happens in the module that uses the mako template.